README.md

```diff
@@ -1,6 +1,8 @@
 # console
 
-[![Build Status](https://travis-ci.org/containerd/console.svg?branch=master)](https://travis-ci.org/containerd/console)
+[![PkgGoDev](https://pkg.go.dev/badge/github.com/containerd/console)](https://pkg.go.dev/github.com/containerd/console)
+[![Build Status](https://github.com/containerd/console/workflows/CI/badge.svg)](https://github.com/containerd/console/actions?query=workflow%3ACI)
```

Comment:
The workflow name for this project is `Go`, not `CI`. Not sure why that is, either `CI` needs to be changed here or rename in `ci.yml` to `CI`. This produces 404

Reply:
I updated ci.yml and renamed to CI, which is what we use in the other repositories.

There's probably some simplification possible in GHA (swap some steps out for `containerd/project-checks@v1`), but I'll look at those in follow ups across the board (it's not really urgent)